src/components/ContactDetails.vue

```diff
@@ -502,6 +509,12 @@ export default {
 		contact() {
 			return this.$store.getters.getContact(this.contactKey)
 		},
+
+		excludeFromBirthdayLabel() {
+			return this.localContact.vCard.hasProperty(this.excludeFromBirthdayKey)
+				? 'Include contact to Birthday Calendar'
```

Comment:
You need to translate the strings with the `t` function:
```suggestion
				? t('contacts', 'Include contact to Birthday Calendar')
```